crates/notion-core/src/error/details.rs

```diff
@@ -16,7 +16,7 @@ an issue at https://github.com/notion-cli/notion/issues with the details!";
 const PERMISSIONS_CTA: &'static str =
     "Please ensure you have correct permissions to the Notion directory.";
 
-#[derive(Debug, Fail)]
+#[derive(Debug, Fail, PartialEq)]
```

Comment:
Adding `PartialEq` lets us check against this in tests! …though not without some finagling, as you can see below. (#334 would simplify a lot of that, however.)